src/src/auth/auth.service.ts

```diff
@@ -87,6 +100,33 @@ export class AuthService {
         }
     }
 
-    /*public async sendOtpService(mobile, otp) {
-    }*/
+    public async sendOtpService(mobileNo, reason, otp) {
+
+        console.log("mobileNo", mobileNo)
+        console.log("otp", otp)
+
+        //let msg = `OTP for ${reason} is ${otp}`
+        let msg =  `प्रिय प्रेरक, प्रगति कैम्प आयोजित करने के लिए कम से कम 8 RSOS पंजीकृत किशोरियों का होना आवश्यक है। कृपया 30 दिसंबर तक एजी मोबिलाइजेशन टूल पर सभी पंजीकृत किशोरियों की जानकारी साझा करें। अपंजीकृत एजी विकल्प को एजी मोबिलाइजेशन टूल पर अक्षम कर दिया गया है I FEGG`
```

Comment:
Msg we need is this

`नमस्ते, प्रगति प्लेटफॉर्म पर सत्यापन/लॉगिन के लिए आपका ओटीपी {#OTP#} है। FEGG`